src/controllers/events.ts

```diff
@@ -0,0 +1,14 @@
+import { Events, EventAttributes, URLAttributes, UserAttributes } from '../db'
+
+export const createEvent = async (url: URLAttributes, req, user: UserAttributes | undefined) => {
+    const event = Events.create({
+        code: url.code,
+        fromIP: req.headers['x-forwarded-for'] || req.connection.remoteAddress,
+        fromURL: req.query.refer || null,
+        userId: user ? user.id : undefined
+    })
+    if (!event) {
+        throw new Error('Could not create event')
```

Comment:
do not throw, just capture in Raven
this should not make the app crash, this is not critical functionality

Reply:
or catch it later ( i have commented below where you can catch)